src/constants/queryKeys.ts

```diff
@@ -69,6 +69,20 @@ const QUERY_KEYS = {
       'historicalPrices',
       { networkId, id },
     ],
+    Joins: {
+      QueryJoin: (
+        amountsIn: Ref<unknown>,
+        hasFetchedPoolsForSor: Ref<unknown>,
+        isSingleAssetJoin: Ref<unknown>
+      ) => [
+        POOLS_ROOT_KEY,
+        'query',
+        'join',
+        amountsIn,
+        hasFetchedPoolsForSor,
+        isSingleAssetJoin,
```

Comment:
I believe this has to be in an object like all the other keys.
```suggestion
        {
          amountsIn,
          hasFetchedPoolsForSor,
          isSingleAssetJoin,
        }
```